content/acra/configuring-maintaining/general-configuration/acra-server/encryptor-config.md

```diff
@@ -0,0 +1,450 @@
+[Read more](/acra/acra-in-depth/data-structures/) about crypto envelopes.
+
+### **reencrypting_to_acrablocks**
+
+Required: `false`
+
+Type: `boolean`
+
+Default value: `false`
+
+
+Description: turns on re-encryption AcraStructs generated on application side to AcraBlock before passing it to database. 
+It replaces less performant AcraStructs to [more performant AcraBlocks](/acra/configuring-maintaining/optimizations/acrastructs_vs_acrablocks/).
+
+If application generates AcraStructs, writes once and reads often then `reencrypting_to_acrablocks: true` makes slower 
+writes due to decryption AcraStructs generated on application side and encrypting into AcraBlocks on AcraServer side, 
+but significantly improves read operations performance due to faster decryption KEK with symmetric cryptography in comparison
+with asymmetric.
+
+If application doesn't generate AcraStructs and rely only on transparent encryption on AcraServer side then we suggest
+turn it off. AcraServer will not try to recognize AcraStruct on every AcraBlock matching failure and slightly improve
+performance for write operations.
+
+## **schemas**
+
+This section defines information about table's columns should be processed by AcraServer. This field is array
+with items:
+```
+schemas:
+  - table: <string>
+    columns: <array of strings>
+    encrypted: <array of structured items>
+  - table: <string>
+    columns: <array of strings>
+    encrypted: <array of structured items>
+```
+
+### **table**
+
+Required: `true`
+
+Type: `string`
+
+Description: table for which should be applied security controls.
+{{< hint warning >}}
+Keep in mind that till 0.93.0 AcraServer processes table name as case-sensitive identifier
+{{</ hint  >}}
+
+### **columns**
+
+Required: `false`
+
+Type: `array of strings`
+
+Description: defines all table's columns in order how they were declared in the database. AcraServer doesn't require
+access to the database or configuring access to database. So it can't get database's metadata with tables' schemas
+and works fully as transparent SQL proxy. 
+
+So, it's important to declare correct order of columns due to it is only one way how to process queries like 
+`SELECT * FROM table1` or `INSERT INTO table1 VALUES (...)`. In first query it helps to understand how much and 
+which columns expect from the database, how to expand `*` value. In the second query it helps support `INSERT` command without explicitly 
+declared columns due to omitted section between table name `table1` and `VALUES` as it could be 
+`INSERT INTO table1 (column1, column2) VALUES(<value1>, <value2>)` with explicitly declared columns and order. 
+
+Without declared columns AcraServer **will not** support these kinds of queries.
+
+### **encrypted**
+
+This section declares security controls like `encryption`, `searchable encryption`, `masking`, `tokenization` on per-column basis for specified
+table.
```

Comment:
here and above: why these are different lines? 147 and 148. Are these extra `\n` necessary?

Reply:
Usually, I try to keep lines not more than 100 characters. But not strictly. It is easier to read and maintain in comparison with long-long lines >100 characters
![image](https://user-images.githubusercontent.com/6237700/162181546-2f0eae70-0491-4364-9b84-e2b6a396568e.png)

Reply:
okay.jpg